Euroleague-Graduate/Euroleague-Graduate/TurkishLeagueTableViewCell.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  TurkishLeagueTableViewCell.swift
+//  Euroleague-Graduate
+//
+//  Created by Modeso on 7/24/17.
+//  Copyright © 2017 Modeso. All rights reserved.
+//
+
+import UIKit
+import  SDWebImage
+
+class TurkishLeagueTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var homeImageView: UIImageView!
+    
+    @IBOutlet weak var awayImageView: UIImageView!
+    
+    @IBOutlet weak var resultsLabel: UILabel!
+    
+    @IBOutlet weak var detailLabel: UILabel!
+    
+    var game: GameData? {
+        didSet{
+            updateUI()
+        }
+    }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+    }
+    
+    private func updateUI() {
+        if game != nil {
```

Comment:
i had used the guard even so i still don't understand it very mush as i search for it because the article link is not working